validator/api/src/main/java/tech/pegasys/teku/validator/api/required/SyncingStatus.java

```diff
@@ -82,6 +90,8 @@ public String toString() {
         + isSyncing
         + ", isOptimistic="
         + isOptimistic
+        + ", elOffline="
+        + elOffline
```

Comment:
whe should migrate this to `MoreObjects.toStringHelper(this)` (check other classes)

Reply:
Done